### PR TITLE
9: Deactivate users.

### PR DESCRIPTION
**Pull request by robherba**
**Steps to test**
- Create a new user.
- Check if the new VBO action is executed successfully.

**DoD**
- In users view (created in a previous ticket) ensure we have a VBO action to enable/disable users.

[issue](https://github.com/ManatiCR/manatime/issues/9)

### CODE REVIEW

File: modules/features/manatime_feature_users/manatime_feature_users.module [hunk 1]
```diff
@@ -5,3 +5,41 @@
...
 include_once 'manatime_feature_users.features.inc';
+
+/**
+ * Implements hook_action_info().
+ */
+function manatime_feature_users_action_info() {
+  return array(
+    'manatime_feature_users_action_enable' => array(
+      'type' => 'entity',
+      'label' => t('Enable users'),
```
Comment (kporras07) [inline modules/features/manatime_feature_users/manatime_feature_users.module]: Active Users

File: modules/features/manatime_feature_users/manatime_feature_users.module [hunk 2]
```diff
@@ -5,3 +5,41 @@
...
+      'behavior' => array('changes_property'),
+      'configurable' => FALSE,
+      'vbo_configurable' => FALSE,
+      'triggers' => array('any'),
+    ),
+    'manatime_feature_users_action_disable' => array(
+      'type' => 'entity',
+      'label' => t('Disable users'),
```
Comment (kporras07) [inline modules/features/manatime_feature_users/manatime_feature_users.module]: Deactive Users

**Comment by kporras07** [comment]
You forgot to recreate the view.

**Comment by robherba** [comment]
Ok @kporras07, changes has been applied.

File: modules/features/manatime_feature_users/manatime_feature_users.module [hunk 3]
```diff
@@ -5,3 +5,41 @@
...
 include_once 'manatime_feature_users.features.inc';
+
+/**
+ * Implements hook_action_info().
+ */
+function manatime_feature_users_action_info() {
+  return array(
+    'manatime_feature_users_action_enable' => array(
+      'type' => 'entity',
+      'label' => t('Active Users'),
```
Comment (kporras07) [inline modules/features/manatime_feature_users/manatime_feature_users.module]: Activate Users (typo)

**Comment by robherba** [comment]
ok @kporras07

**Comment by kporras07** [comment]
Assuming Wercker passes, :shipit:

**Pull request merged**